vmdb/app/helpers/container_node_helper/textual_summary.rb

```diff
@@ -15,4 +15,12 @@ def textual_group_properties
   def textual_name
     {:label => "Name", :value => @record.name}
   end
+
+  def textual_creation_timestamp
+    {:label => "Creation Timestamp", :value => @record.creation_timestamp}
```

Comment:
Do times have to presented in a nice way for the UI, especially with respect to time zones?  cc @h-kataria